bitnami/discourse/2/debian-11/rootfs/opt/bitnami/scripts/libdiscourse.sh

```diff
@@ -249,6 +249,7 @@ discourse_create_conf_file() {
     discourse_conf_set "redis_host" "$DISCOURSE_REDIS_HOST"
     discourse_conf_set "redis_port" "$DISCOURSE_REDIS_PORT_NUMBER"
     discourse_conf_set "redis_password" "$DISCOURSE_REDIS_PASSWORD"
+    discourse_conf_set "redis_use_ssl" "${DISCOURSE_REDIS_USE_SSL:-false}"
```

Comment:
Are we sure this is the way defaults are set in the `libdiscourse.sh` file?

Reply:
Default in Discourse of property "redis_use_ssl" is "false" (and that's why this PR is necessary: it runs with disabled SSL support for Redis.). That's why this extension is not breaking anything at all.

Default setting of discourse see: https://github.com/discourse/discourse/blob/e6dba65f187b932f6ea8171ea35454c909aeaa10/config/discourse_defaults.conf#L152

Reply:
I'm not worried about that. I'm puzzled on why we're setting a default via bash as well (`${DISCOURSE_XYZ:-default}` when all of the other options aren't doing that.

Isn't it enough to just set the variable? Doesnt `discourse_conf_set` rewrite the default value (set in the default config) only if you pass a non-empty value as an argument?

Reply:
Having a look at the implementation of "discourse" property handling, the current implementation would actually only set the 'ssl' property of Redis config to true, if the "redis_use_ssl" property would be "true". 

I.e. it would actually work, when not implementing a default. Nevertheless it looks not like a good idea to write a property entry "redis_use_ssl=" into the "discourse.conf" file, as this is invalid (and one would have to checkout Discourse code to find out, how this would be handled).

If you really dislike the default value, we would have to wrap this line within a  `if [ -n "$DISCOURSE_REDIS_USE_SSL" ]; then`

Reply:
Nono, I honestly don't mind. I was more wondering on whether this fits the style of this file 😅